src/connectors/jamendo.js

```diff
@@ -10,6 +10,10 @@ Connector.currentTimeSelector = '.hidden-xs > .js-player-position';
 
 Connector.durationSelector = '.hidden-xs > .js-player-duration';
 
-Connector.playButtonSelector = '.js-player-play-pause > .icon-play';
+Connector.isPlaying = () => {
+	return $('.js-player-play-pause > .icon-pause').length !== 0;
+};
 
-Connector.trackArtSelector = '.js-full-player-cover-img';
```

Comment:
Wouldn't changing this selector to `.hero-cover img` be better?

Reply:
That also works. I tried to get a "src" from image. I guess both works.

What do you prefer more? If you prefer that one more, please tell me. I will take it over.